content/team.md

```diff
@@ -21,6 +21,7 @@ Jan is, together with Eduard intimately involved in organizing DotNetFriday. He
 
 ## Erwin Staal
 ![Jan de Vries](/img/speakers/erwin-staal.jpg)
```

Comment:
Nog even de alt van de image aanpassen en is de afbeelding ook al toegevoegd?

Reply:
Ah dat deel stond er al dus niet naar gekeken. Afbeelding is er al, gecheckt op de live site. Ik pas de alt ff aan